packages/@expo/cli/src/start/doctor/dependencies/ensureDependenciesAsync.ts

```diff
@@ -96,18 +99,18 @@ function wrapForTerminal(message: string): string {
 
 /** Create the bash install command from a given set of packages and settings. */
 export function createInstallCommand({
-  isYarn,
+  manager,
   packages,
 }: {
-  isYarn: boolean;
+  manager: PackageManager.NodePackageManager;
   packages: {
     file: string;
     pkg: string;
     version?: string | undefined;
   }[];
 }) {
   return (
-    (isYarn ? 'yarn add' : 'npm install') +
+    (manager === 'yarn' ? `${manager} add` : `${manager} install`) +
```

Comment:
I think `pnpm` also uses add:
![image](https://user-images.githubusercontent.com/1203991/168055200-00e50d9a-dfbf-4fa6-b150-8c9f5a813ae9.png)

Reply:
Looks like it also uses `install`